dist/controller.py

```diff
@@ -48,24 +48,20 @@ def main():
     print("Extracting arguments from command line.")
     env_vars = extract_args()
 
-    # Clean environment before mining
     print("Data mining for Living Documentation")
-    run_script('clean_env_before_mining.py', env_vars)
+
+    clean_environment()
```

Comment:
Here, we have to change the way how we manage env variables.
- moving them to the controller
- setting them into local env copy
Then, we will be able to call all scripts in this way.

I am trying to say that this is half done solution.